evaluation/run-10/per-file-evaluation.md

```
@@ -0,0 +1,209 @@
+### auto-summarize.js (3 spans)
+
+| Rule | Result |
+|------|--------|
+| All rules | ALL PASS |
+
+### journal-manager.js (2 spans)
+
+| Rule | Result | Notes |
+|------|--------|-------|
+| Most rules | PASS | |
+| CDQ-006 | **Advisory** | `(commit.message || '').split('\n')[0]` — `.split()` to get first line. Practically trivial on short strings. Advisory per run-9 methodology. |
```

markdownlint-cli2 (0.21.0)

[warning] Reversed link syntax
('\n')[0]

(MD011, no-reversed-links)